.golangci.yml

```diff
@@ -35,12 +35,14 @@ linters:
     - golint            # deprecated by Go team
     - gomnd             # some unnamed constants are okay
     - ifshort           # deprecated by author
+    - inamedparam       # convention is not followed
     - interfacer        # deprecated by author
     - ireturn           # "accept interfaces, return structs" isn't ironclad
     - lll               # don't want hard limits for line length
     - maintidx          # covered by gocyclo
     - maligned          # readability trumps efficient struct packing
     - nlreturn          # generous whitespace violates house style
+    - nonamedreturns    # named returns are fine; it's *bare* returns that are bad
```

Comment:
Don't we have a tonne of named errors in deferred statements, or is this what you mean by "bare" returns?

Reply:
Yeah we do - that's why this linter is disabled. In downstream projects the config isn't always up to date - some still enable selected linters instead of enabling all and disabling ones we don't want.

Reply:
doh, misread this is the linters.disable list